tests/Providers/Trustpilot/ServiceActionsTest.php

```diff
@@ -0,0 +1,128 @@
+
+        $businessId = 'foo-bar-123';
+        $action->setMerchantId($businessId);
+
+        $this->assertEquals('foo-bar-123', $action->getMerchantId());
+    }
+
+    /**
+     * Making sure the invite method to make sure it returns the response as expected.
+     *
+     * @return array|mixed
+     * @throws \PodPoint\Reviews\Exceptions\ValidationException
+     */
+    public function testInvite()
+    {
+        $options = [
+            'referenceNumber' => 'reference-number-321',
+            'consumerEmail' => 'customer@email.com',
+            'consumerName' => 'Customer Name',
+        ];
+
+        $response = $this->getMockedResponse('{"status": "200", "message": "accepted"}');
+        $apiClient = $this->getMockedApiClient();
+        $apiClient->shouldReceive('sendRequest')->withAnyArgs()->andReturn($response);
+
+        $action = new MerchantActions($apiClient);
+        $action->setMerchantId('foo-bar-321');
+
+        $inviteResponse = $action->invite($options);
+
+        $expectedResult = [
+            "status" => "200",
+            "message" => "accepted",
+        ];
+
+        $this->assertEquals($expectedResult, $inviteResponse);
+    }
+
+    /**
+     * Making sure the getReviews method to make sure it returns the response as expected.
+     *
+     * @return array|mixed
+     * @throws \PodPoint\Reviews\Exceptions\ValidationException
+     */
+    public function testGetReviews()
+    {
+        $options = [];
+
+        $response = $this->getMockedResponse('{"status": "200", "message": "successful"}');
+        $apiClient = $this->getMockedApiClient();
+        $apiClient->shouldReceive('sendRequest')->withAnyArgs()->andReturn($response);
+
+        $action = new MerchantActions($apiClient);
+        $action->setMerchantId('foo-bar-321');
+
+        $getReviewsResponse = $action->getReviews($options);
+
+        $expectedResult = [
+            "status" => "200",
+            "message" => "successful",
+        ];
+
+        $this->assertEquals($expectedResult, $getReviewsResponse);
+    }
+
+    /**
+     * Making sure the findReview method to make sure it returns the response as expected.
+     *
+     * @return array|mixed
+     * @throws \PodPoint\Reviews\Exceptions\ValidationException
+     */
+    public function testFindReview()
+    {
+        $reviewId = 'review-id-123';
+
+        $response = $this->getMockedResponse('{"status": "200", "message": "successful"}');
+        $apiClient = $this->getMockedApiClient();
+        $apiClient->shouldReceive('sendRequest')->withAnyArgs()->andReturn($response);
+
+        $action = new MerchantActions($apiClient);
+        $action->setMerchantId('foo-bar-321');
+
+        $findReviewResponse = $action->findReview($reviewId);
+
+        $expectedResult = [
+            "status" => "200",
+            "message" => "successful",
+        ];
+
+        $this->assertEquals($expectedResult, $findReviewResponse);
```

Comment:
I think here we can do Response::HTTP_OK instead of 200 same for everywhere else

Reply:
This is not an actual response, just making sure the response {"status": "200", "message": "successful"} is parsed into an array. Not sure if needed here.